build-package.sh

```diff
@@ -984,6 +988,12 @@ termux_step_make() {
 		else
 			make -j $TERMUX_MAKE_PROCESSES $QUIET_BUILD ${TERMUX_PKG_EXTRA_MAKE_ARGS}
 		fi
+	elif test -f build.ninja || [ ! -z "$TERMUX_PKG_EXTRA_NINJA_ARGS" ]; then
```

Comment:
Currently assumes there's no Makefile lying around if build.ninja's there, can change it to only do one or the other if that's preferred.